docker-compose.yml

```diff
@@ -25,9 +25,9 @@ services:
       context: .
       cache_from:
         - ${REPO}:nanoarrow-${PLATFORM}
+      cache_to:
+        - ${REPO}:nanoarrow-${PLATFORM}
       dockerfile: ci/docker/${PLATFORM}.dockerfile
-      args:
-        PLATFORM: ${PLATFORM}
```

Comment:
No image used this as far as I could see but feel free to revert the change if I missed something :)